### DISCUSSION TITLE
我换了新的API Keys，现在不能对话

### DISCUSSION THREAD

**Discussion by steven731** [community/community#51315] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

以前的KEY过期了，换了新的KEY后出现在下面的揭示：

助手
API 请求失败：

Failed to fetch
word count: 8, token estimate: 27

**Comment by steven731** [comment]
还是网络问题，换了一个梯子才能连接上。

**Comment by queenofcorgis** [comment]
Hi @steven731, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one. 

Additionally,  You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects.
